code/modules/food&drinks/food/snacks_cake.dm

```diff
@@ -1,139 +1,163 @@
 
-/obj/item/weapon/reagent_containers/food/snacks/sliceable/store/carrotcake
+/obj/item/weapon/reagent_containers/food/snacks/store/cake
+	slice_path = /obj/item/weapon/reagent_containers/food/snacks/cakeslice/plain
+	slices_num = 5
+	bitesize = 3
+
+/obj/item/weapon/reagent_containers/food/snacks/cakeslice
+	trash = /obj/item/trash/plate
+	bitesize = 3
+
+/obj/item/weapon/reagent_containers/food/snacks/store/cake/plain
+	name = "vanilla cake"
+	desc = "A plain cake, not a lie."
+	icon_state = "plaincake"
+	custom_food_type = /obj/item/weapon/reagent_containers/food/snacks/customizable/cake
+	list_reagents = list("nutriment" = 5, "vitamin" = 5)
+
+/obj/item/weapon/reagent_containers/food/snacks/cakeslice/plain
+	name = "vanilla cake slice"
+	desc = "Just a slice of cake, it is enough for everyone."
+	icon_state = "plaincake_slice"
+	filling_color = "#FFD700"
+
+/obj/item/weapon/reagent_containers/food/snacks/store/cake/carrot
 	name = "carrot cake"
 	desc = "A favorite desert of a certain wascally wabbit. Not a lie."
 	icon_state = "carrotcake"
-	slice_path = /obj/item/weapon/reagent_containers/food/snacks/carrotcakeslice
+	slice_path = /obj/item/weapon/reagent_containers/food/snacks/cakeslice/carrot
 	slices_num = 5
-	list_reagents = list("nutriment" = 20, "oculine" = 10, "vitamin" = 5)
+	list_reagents = list("oculine" = 10, "vitamin" = 10)
```

Comment:
The cake is literally not filling at all? That seems silly. Even if it is vanilla base + carrot add-on, it should at least have 1u of nutriment.